python/lsst/afw/table/_syntax.py

```diff
@@ -202,3 +202,70 @@ def processArray(a):
         else:
             d[name] = processArray(self.get(schemaItem.key))
     return d
+
+def BaseCatalog_asAstropy(self, cls=None, copy=False, unviewable="copy"):
```

Comment:
I don't really like `asAstropy`. I'd prefer `as_astropy_table` but I understand that camel case is king. I don't really care if it's an internal method.